public/app/index.ts

```diff
@@ -1,4 +1,6 @@
-import app from './app';
+declare let __webpack_public_path__: string;
+__webpack_public_path__ = window.public_cdn_path;
```

Comment:
is this used?

Reply:
Yes this is the key part of telling webpack where to load files

Reply:
Without this all the dynamic webpack chunks are loaded from localhost